### PR TITLE
Check gen sandbox

### PR DESCRIPTION
**Pull request by ruslansennov**
not intended to be merged

### CODE REVIEW

File: javaslang/src-gen/main/java/javaslang/Tuple.java [hunk 1]
```diff
@@ -1,8 +1,47 @@
...
+        J   L  |   L                     J    |
+         L  J  L    \                    L    \
+         |   L  ) _.'\                    ) _.'\
+         L    \('`    \                  ('`    \
+          ) _.'\`-....'                   `-....'
+         ('`    \
+          `-.___/
+
+
+*/
```
Comment (danieldietrich) [inline javaslang/src-gen/main/java/javaslang/Tuple.java]: Woohooo! :-)))

File: run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ev
+
+mvn test -B
+
+# Break CI build on PRs where committed differ from generated sources
```
Comment (danieldietrich) [inline run-tests.sh]: Yay!

File: javaslang/src-gen/main/java/javaslang/Tuple.java (hunk 1 above)
Comment (danieldietrich) [inline javaslang/src-gen/main/java/javaslang/Tuple.java]: I'm really tempted to merge this PR :-)

Comment (ruslansennov) [inline javaslang/src-gen/main/java/javaslang/Tuple.java]: so cute :)

Comment (danieldietrich) [inline javaslang/src-gen/main/java/javaslang/Tuple.java]: Could be our mascot ;)

**Pull request closed without merging**